mqtt/Dockerfile

```diff
@@ -1,5 +1,4 @@
-FROM node:boron-alpine
-MAINTAINER Mainflux
+FROM node:10.15.1-alpine
```

Comment:
Why MQTT Dockerfile is updated as a part of Elm UI PR? If it needs an update, I'd prefer a separate PR for that.

Reply:
I did this, to use opportunity. I was correcting stuff in parallel. Leave this change here please.